### PR TITLE
updated doc with generate request status

### PR DESCRIPTION
**Pull request by NoSkillGirl**
Signed-off-by: NoSkillGirl <singhpooja240393@gmail.com>

closes https://github.com/kyverno/website/issues/380

### CODE REVIEW

File: content/en/docs/Writing policies/generate.md [hunk 1]
```diff
@@ -26,6 +26,18 @@ Kubernetes has many default resource types even before considering CustomResourc
...
+
+```sh
+kubectl get generaterequests.kyverno.io -A
+```
+
+Generate request status can have four values:
+`Completed` - the generate request controller created resources defined in the policy
+`Failed` - the generate request controller failed to process the rules
+`Pending` - the request is yet to be processed or resource has not been created
+`Skip` - the generate request controller has skipped the creation of resource
```
Comment (realshuting) [inline content/en/docs/Writing policies/generate.md:39]: Is the `Skip` only be used when the source resource already exists in the cluster while the policy does not have a label/annotation selector defined?

Comment (NoSkillGirl) [reply content/en/docs/Writing policies/generate.md:39]: for now yes. But skip can be used as a generic status. And when we do so this can be updated.

Comment (realshuting) [reply content/en/docs/Writing policies/generate.md:39]: I don't see other use cases that we need to mark as `Skip`. If the source does not match, why bother create a GR and mark as `Skip`?

Comment (NoSkillGirl) [reply content/en/docs/Writing policies/generate.md:39]: please check [this](https://github.com/kyverno/kyverno/pull/2657) PR 
the GR is created before and then processed for resource creation. This is skipped [here](https://github.com/kyverno/kyverno/blob/5195bc5bf298a9d81252c349e4231a7b91cbaec7/pkg/generate/generate.go#L297), without any error.

Comment (realshuting) [reply content/en/docs/Writing policies/generate.md:39]: Isn't this the case I described above?
>Is the Skip only be used when the source resource already exists in the cluster while the policy does not have a label/annotation selector defined?

Comment (NoSkillGirl) [reply content/en/docs/Writing policies/generate.md:39]: Yes. And when the preexisting resource is updated the new resource is not generated.

Comment (chipzoller) [reply content/en/docs/Writing policies/generate.md:39]: Is this resolved?

**Comment by chipzoller** [comment]
@NoSkillGirl can you please help close this PR out by addressing comments?

File: content/en/docs/Writing policies/generate.md (hunk 1 above)
Comment (chipzoller) [reply content/en/docs/Writing policies/generate.md:39]: Is this resolved?

**Comment by chipzoller** [comment]
@NoSkillGirl Please help close this out.

**Comment by chipzoller** [comment]
Any more work need to be done here?

**Comment by chipzoller** [comment]
Ping @realshuting . I need to get to work on the 1.6.0 docs this weekend and need help getting to a "clean" state first.

**Comment by chipzoller** [comment]
No response. Incorporating this into forthcoming update.

**Pull request closed without merging**